plugins/flytekit-kf-pytorch/flytekitplugins/kfpytorch/task.py

```diff
@@ -455,6 +474,14 @@ def fn_partial():
         except SignalException as e:
             logger.exception(f"Elastic launch agent process terminating: {e}")
             raise IgnoreOutputs()
+        except Exception as e:
```

Comment:
Can we do this in another PR please?
Is my understanding correct that any actual user error would have been a `ChildFailedError`? Do you have an example of an exception that isn't yet handled?

Reply:
I've seen `RendezvousError` happening. Typically it happens when there is a much earlier ChildError from another group. Pod A has a ChildError, Pod B has a RendezvousError, and if Pod B's error file takes precedence, then we end up preventing a task restart :( 

I have this change in our local environment (via monkey patching `elastic_launch`), because with the current behavior of task outputs overwriting each other, these exceptions are preventing task restarts. 

I agree that this can belong to a separate PR. Btw, if we fix the task error file issue and pick the earliest, then the impact of this is smaller (but I still think treating these as non-recoverable user errors is a surprising behavior).

Reply:
Can you please open another PR for this? I think we can merge this much faster than proper treatment of multiple error files.

Reply:
@fg91 Thanks for reviewing the PR 😄